### ISSUE TITLE
[@types/colors] : Missing string property

### ISSUE THREAD

**Issue by WanFoxOne** [DefinitelyTyped/DefinitelyTyped#21919]
- [X] I recommend an improvement for the `@types/colors` package.

This property is missing on the file ___index.s.ts___
```Typescript
declare global {
    interface String {
        [...]
        custom: object; // <-- Missing property
    }
}
```

- Authors: @WanFoxOne

**Comment by DABH** [comment]
TS definitions for colors are moving to the colors repo: https://github.com/Marak/colors.js/tree/develop .  Please open a PR there and I'll take a look!  Thanks!

**Comment by DABH** [comment]
Please close this issue -- it appears to be solved as of `colors@1.2.0` (colors bundles its own types now, `@types/colors` not required).  Thanks!
